docs/workforce-management-framework/upstream-1/master.adoc

```diff
@@ -5,34 +5,41 @@ include::topics/shared/attributes.adoc[]
 :toclevels: 5
 :last-update-label!:
 
+image::../../shared/images/logo-blue.png[logo]
+
 [[Raincatcher]]
 = {WFM-RC-NameLong}
```

Comment:
The line of code that includes the image breaks CCUTIL for me - I've to comment it out to build it.

Best to leave this in for now and I'll think of a fix. Paul suggested to add an if statement so as to avoid such issues however, I need to think it through as I'm not sure how best to implement his proposed fix.